decidim-system/db/seeds.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-if !Rails.env.production? || ENV["SEED"]
+if !Rails.env.production? || ENV.fetch("SEED", nil)
   print "Creating seeds for decidim-system...\n" unless Rails.env.test?
 
-  password = ENV["DECIDIM_SYSTEM_USER_PASSWORD"] || "decidim123456"
+  password = ENV.fetch("DECIDIM_SYSTEM_USER_PASSWORD", nil) || "decidim123456"
```

Comment:
I think better would be:
```suggestion
  password = ENV.fetch("DECIDIM_SYSTEM_USER_PASSWORD", "decidim123456")
```